src/Sulu/Component/CustomUrl/Manager/CannotDeleteCurrentRouteException.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\CustomUrl\Manager;
+
+use Sulu\Bundle\ContentBundle\Document\RouteDocument;
+use Sulu\Component\CustomUrl\Document\CustomUrlDocument;
+use Sulu\Component\Rest\Exception\RestException;
+
+/**
+ * Thrown when a current route will be deleted.
+ */
+class CannotDeleteCurrentRouteException extends RestException
```

Comment:
What do you mean by current route? And would rename that to something without a verb, to follow the naming of other exceptions. Maybe `RouteNotRemovableException` or something similar?